bluemix/authentication/vpc/vpc.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 const (
-	DefaultMetadataServiceVersion = "2021-09-30"
+	DefaultMetadataServiceVersion = "2022-06-30"
```

Comment:
Please see the recommended change here https://github.com/IBM-Cloud/ibm-cloud-cli-sdk/issues/327#issuecomment-1171427231